scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -723,10 +733,7 @@ def create_lvm(self, index, metadata_dev):
     return True
 
 def calc_lvm_min_size(self, lv_path, lvm_min_size):
-    if self.k8 == 'K8':
-        cmd = f"lsblk --noheadings --bytes {lv_path} | " "awk '{print $4}'"
-    else:
-        cmd = f"lvs {lv_path} -o LV_SIZE --noheadings --units b --nosuffix"
+    cmd = f"lsblk --noheadings --bytes {lv_path} | " "awk '{print $4}'"
```

Comment:
I see that there is a parameter for disk size in confstore key, we should usethat instead of reading it from filesystem

Reply:
disk size is not provided in /etc/cortx/cluster.conf which is created in configmap.
disk size is provided in solution.yaml but is not included in /etc/cortx/cluster.conf which is passed to mini prov phases. So currently we can't use disk size unless it is available in /etc/cortx/cluster.conf